src/identify_client.cpp

```diff
@@ -290,7 +290,7 @@ namespace
 			|| ((lhs.id[0] == rhs.id[0]) && (lhs.id[1] < rhs.id[1]));
 	}
 
-	std::string lookup(fingerprint const& f)
+	static std::string lookup(fingerprint const& f)
```

Comment:
if you meant to give this internal linkage, please put it in the anonymous namespace instead of using ``static``